packages/block-library/src/query/edit/inspector-controls/sticky-control.js

```diff
@@ -8,6 +8,7 @@ const stickyOptions = [
 	{ label: __( 'Include' ), value: '' },
 	{ label: __( 'Exclude' ), value: 'exclude' },
 	{ label: __( 'Only' ), value: 'only' },
+	{ label: __( 'Ignore' ), value: 'ignore' },
```

Comment:
How does this affect the editor? Shouldn't there be some change to the querying done clientside in the editor too?

Reply:
Great question.

In the editor, the sticky posts are already bugged on trunk. For example sticky posts do not show as the first posts when the "include" option is set, which is the default.

Reply:
We should at least try to see if we can match the preview in editor. As @carolinan says it's already buggy and if I remember correctly is about what REST API supports how it maps with WP_Query. That's probably the reason (the mapping) for not having this as a separate option for sticky control.